src/data/equiv/mul_add.lean

```diff
@@ -213,6 +213,23 @@ e.to_equiv.symm_apply_eq
 lemma eq_symm_apply (e : M ≃* N) {x y} : y = e.symm x ↔ e y = x :=
 e.to_equiv.eq_symm_apply
 
+section
+variables [mul_one_class M] [mul_one_class N] [mul_one_class P]
+
+@[to_additive] lemma eq_comp_symm (e : M ≃* N) (f : N →* P) (g : M →* P) :
+  ⇑f = g ∘ e.symm ↔ f ∘ e = g := e.to_equiv.eq_comp_symm f g
+
+@[to_additive] lemma comp_symm_eq (e : M ≃* N) (f : N →* P) (g : M →* P) :
+  g ∘ e.symm = f ↔ ⇑g = f ∘ e := e.to_equiv.comp_symm_eq f g
+
+@[to_additive] lemma eq_symm_comp (e : M ≃* N) (f : P →* M) (g : P →* N) :
+  ⇑f = e.symm ∘ g ↔ e ∘ f = g := e.to_equiv.eq_symm_comp f g
+
+@[to_additive] lemma symm_comp_eq (e : M ≃* N) (f : P →* M) (g : P →* N) :
+  e.symm ∘ g = f ↔ ⇑g = e ∘ f := e.to_equiv.symm_comp_eq f g
```

Comment:
These are true more generally as:
```suggestion

@[to_additive] lemma eq_comp_symm (e : M ≃* N) (f : N → P) (g : M → P) :
  f = g ∘ e.symm ↔ f ∘ e = g := e.to_equiv.eq_comp_symm f g

@[to_additive] lemma comp_symm_eq (e : M ≃* N) (f : N → P) (g : M → P) :
  g ∘ e.symm = f ↔ g = f ∘ e := e.to_equiv.comp_symm_eq f g

@[to_additive] lemma eq_symm_comp (e : M ≃* N) (f : P → M) (g : P → N) :
  f = e.symm ∘ g ↔ e ∘ f = g := e.to_equiv.eq_symm_comp f g

@[to_additive] lemma symm_comp_eq (e : M ≃* N) (f : P → M) (g : P → N) :
  e.symm ∘ g = f ↔ g = e ∘ f := e.to_equiv.symm_comp_eq f g
```
which avoids the need for the `mul_one_class` above